content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -571,9 +571,11 @@ If filing a bug, please include detailed information about how to reproduce the
 
 We have a lot of features in our roadmap. An abbreviated high level list is included below, but we encourage you to view our [roadmap project](https://github.com/orgs/kubernetes/projects/8) and help us make Windows support better by [contributing](https://github.com/kubernetes/community/blob/master/sig-windows/).
 
-### CRI-ContainerD
+### CRI-ContainerD (alpha in v1.18)
```

Comment:
@PatrickLang this will break incoming links at least twice: once when adding the feature state, then again when it changes.

Use a [feature state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) instead.

Reply:
thanks! fixed